### ISSUE TITLE
I tried this with beta8 and seems to be a callback issue

### ISSUE THREAD

**Issue by 4deeptech** [leastprivilege/AspNet5TemplateExternalAuthentication#1]
I believe I have everything configured correctly as far as Google is concerned but there seems to be an issue with the callback.

app.UseGoogleAuthentication(new GoogleOptions
            {
                ClientId = "[my id]",
                ClientSecret = "[mysecret]",

                AuthenticationScheme = "Google",
                SignInScheme = "Cookies",
                CallbackPath = new PathString("/signin-google2")  <--sets the callback url
            });

I just set it to that so the error message Google gives me confirms that it sees that I can control and set the callback.  The issue is that I'm not sure what the callback url should be for this sample. Is the sample missing this?

**Comment by leastprivilege** [comment]
I set up this redirect URI in the google dev console:

http://localhost:4236/signin-google

**Comment by 4deeptech** [comment]
Yes.  I ended up sorting it all out.  I tried to issue a pull request with a couple minor updates but didn't have permission.  I was going to update the readme to give instructions about the redirect uri and that you have to enable Google+ API for your App/key in the Google developer console or you will get a access_denied error.  Also, The static files support was broken so it was not loading up css content.  The static files reference in project.json needed to be updated to beta8 and I updated the razor reference to beta8 as well.

**Comment by leastprivilege** [comment]
What do you mean with "no permission" ?

**Comment by 4deeptech** [comment]
I cloned the repo locally and got things sorted out.  Then decided to try to submit the changes back and realized I hadn't created a separate branch and had committed to master locally.  I couldn't push those changes back up to master due to permission issues (GitHub for windows errored saying I didn't have permission).  I created a branch locally and changed the google api key back to your original one so mine wouldn't get commited.  After creating my branch and tweaking the key, I committed it locally and tried to create a pull request via GitHub for windows and it too failed saying I did not have permission.  I believe the issue is that I did not fork it and work my changes on the fork.
